java/src/client/facade/FacadeManager.java

```diff
@@ -0,0 +1,65 @@
+package client.facade;
+
+import shared.models.game.ClientModel;
+
+/**
+ * Created by audakel on 9/17/16.
+ */
+public class FacadeManager {
+    /** A reference to the game info in the client model, will update frequently */
+    private ClientModel mClientModel;
```

Comment:
android and java use following Field Naming Conventions
Non-public, non-static field names start with m.

http://stackoverflow.com/questions/2092098/why-do-most-fields-class-members-in-android-tutorial-start-with-m/7072899#7072899